parm/fv3lam_rrfs.xml

```diff
@@ -2528,6 +2546,16 @@
        <scale>3.0</scale>
        </param>
 
+       <param>
+       <shortname>GSD_NCRAIN_ON_HYBRID_LVL</shortname>
+       <pname>SPNCR</pname> 
+       <table_info>NCEP</table_info>
```

Comment:
@jaymes-kenyon This line should be deleted for properly generating SPNCR.

Reply:
Deleted the "table_info" line